pkg/api/annotations/pod.go

```diff
@@ -52,3 +52,11 @@ const PodTaskExecutionRole = "pod.elotl.co/task-execution-role"
 // that containers in a fargate task can assume. All containers
 // in the task assume this role.
 const PodTaskRole = "pod.elotl.co/task-role"
+
+// The PodHealthcheckHealthyTimeout annotation is used to customize the
+// healthcheck timeout for pods. If a pod doesn't have a healthy
+// response to healthcheck probes for greater than healthcheck-timeout
+// the pod will be terminated and restarted according to the pod's
+// restartPolicy.  A healthcheck-timeout equal to zero means the pod
+// will not be terminated due to failing healthchecks.
+const PodHealthcheckHealthyTimeout = "pod.elotl.co/healthcheck-healthy-timeout"
```

Comment:
I think this one is iffy for a few reasons.

1. Putting "pod" in the annotation name has brought up a strange classification issue since we have a 1-1 mapping from pods to cells.

2. Healthchecks could be interpreted a few ways: From the user's perspective, the annotation applies to the pod.  While we're actually health checking the underlying instance the pod is running on, that healtcheck changes the behavior of the pod and is tied to the pod.  For example an elasticsearch pod might might want to turn off health checks so it doesn't get destroyed.

3. From a technical perspective, all the existing annotations apply to cells anyways (instance-type, private IPs, instance profiles).  I don't think this one is very different from that.  I'd rather not create a new naming convention for this annotation when it's in a similar classification to the existing annotations